src/bindings/python/__init__.py

```diff
@@ -7,8 +7,6 @@
 """
 
 from .phrasedml import *
-from pkg_resources import get_distribution
 
-#__version__ = LIBPHRASEDML_VERSION_STRING[1:]
  __version__ = get_distribution('phrasedml').version
 

```

Comment:
At first I thought there was no "__version__" due to some issue with `__init__.py`. But now that phrasedml.py and __init__.py are correctly copied the original version of `__init__.py` seems to be working fine. So I've reverted it.